Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/SingleFileTransfer.cs

```diff
@@ -334,9 +331,9 @@ private void ExecuteSourceOperationMoveOrRename()
                 _logger.NotifyError(BatchContext, msg, new ArgumentException("Failure in moving the source file."));
```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/33bbc16e411f6fa4f5f8086d85e2a83889335206/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/SingleFileTransfer.cs#L205
This can be a static method.